src/dependencytracker.js

```diff
@@ -252,6 +255,20 @@ class DependencyTracker {
         }
       },
     });
+
+    await this.TaskGroupActiveSet.remove({taskId, taskGroupId}, true);
+
+    // check for emptiness of the partition
+    let result = await this.TaskGroupActiveSet.query({taskGroupId}, {limit: 1});
+
+    // We assume this generally won't happen
+    assert(result.entries.length > 0 || !result.continuation, 'hmm... this is bad, see comment in isBlocked()');
```

Comment:
Actually, couldn't we do `new Error(...)` and then add the `taskId` and `taskGroupId` as properties on the error.. that way we'll have them in sentry... it's only a few lines...